### PR TITLE
Isis sidebar modifications for accessibility Fix #6758

### LINKED ISSUE #6758: Isis sidebar needs some modifications for accessibility
I recently went ahead and installed chrome vox plugin and tried it on backend with isis. There are many things that needs to be improved, but the most obvious is the faulty hidden sidebar. So try this:
go to `administrator/index.php?option=com_users&view=users`
and HIDE the sidebar!
now start navigating with the tab
the order after the help button on the toolbar is wrong!
It should be highlighting the search field but instead is highlighting the hidden elements of the sidebar!

How to solve this?
The script that hides/shows the sidebar should also inject `aria-hidden="true"` to the ul element when that is hidden and remove it when the sidebar is viewable.

### PR DESCRIPTION
**Pull request by vinespie**
Adding attributes "aria-hidden" set to "true" and "tabindex" set to "-1" when sidebar is hidden. If visible these attributes are removed.
## Actual result

In admin / contact manager when you navigate with "tab" and the sidebar is hidden, you spend anyway on links and filters.
## Test instructions
1. Apply the patch
2. Go to Admin
3. Go to contact manager 
4. Hide the sidebar
5. Start navigation with the tab
6. After the help button, you must go to search field
7. Show the the sidebar
8. Restart navigation whith the tab
9. You must pass in the sidebar links and filters

### CODE REVIEW

File: media/system/js/html5fallback-uncompressed.js [hunk 1]
```diff
@@ -179,13 +179,11 @@
 				$elem.addClass(self.options.invalidClass);
 				var $labelref = self.findLabel($elem);
 				$labelref.addClass(self.options.invalidClass);
-				$labelref.attr('aria-invalid', 'true');
```
Comment (dgt41) [inline media/system/js/html5fallback-uncompressed.js]: Can I ask why you want to remove this?

**Comment by dgt41** [comment]
@test Success Thanks!

File: media/system/js/html5fallback-uncompressed.js (hunk 1 above)
Comment (vinespie) [inline media/system/js/html5fallback-uncompressed.js]: Sorry @dgt41, is in link with #6573

Comment (dgt41) [inline media/system/js/html5fallback-uncompressed.js]: can you add also the testing instructions from #6573 ?

**Comment by bertmert** [comment]
@test Success.

File: media/system/js/html5fallback-uncompressed.js (hunk 1 above)
Comment (vinespie) [inline media/system/js/html5fallback-uncompressed.js]: ok @dgt41 I added the test instruction to #6573

**Comment by zero-24** [comment]
RTC Thanks! :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7024">issues.joomla.org/joomla-cms/7024</a>.</sub>

**Comment by wilsonge** [comment]
Merged - thanks!

**Pull request closed without merging**